docs/alpine-base-update.md

```diff
@@ -130,7 +130,7 @@ following which is an explanation of each one.
 ```sh
 # Update tools packages
 cd $LK_ROOT/tools
-$LK_ROOT/scripts/update-component-sha.sh --pkg $LK_ROOT/tools/alpine
+$LK_ROOT/scripts/update-component-sha.sh --image $LK_ALPINE
```

Comment:
what's the reason for this change? Does `update-component-sha.sh` no longer take the `--pkg` option?

Reply:
It does, but `tools/alpine` does not have a `build.yml`. To be fair, I have no idea how it worked before, but if this works, and we already are storing the `LK_ALPINE` var, might as well use it.